articles/virtual-desktop/app-attach-setup.md

```diff
@@ -192,12 +192,13 @@ Here's how to add an MSIX or Appx image as an app attach package using the [Az.D
    Your output should be similar to the following output:
 
    ```output
-   CommandType     Name                                            Version    Source
-   -----------     ----                                            -------    ------
-   Function        Get-AzWvdAppAttachPackage                       4.2.0      Az.DesktopVirtualization
-   Function        New-AzWvdAppAttachPackage                       4.2.0      Az.DesktopVirtualization
-   Function        Remove-AzWvdAppAttachPackage                    4.2.0      Az.DesktopVirtualization
-   Function        Update-AzWvdAppAttachPackage                    4.2.0      Az.DesktopVirtualization
+CommandType     Name                                               Version    Source
+-----------     ----                                               -------    ------
+Function        Get-AzWvdAppAttachPackage                          4.2.1      Az.DesktopVirtualization
+Function        Import-AzWvdAppAttachPackageInfo                   4.2.1      Az.DesktopVirtualization
+Function        New-AzWvdAppAttachPackage                          4.2.1      Az.DesktopVirtualization
+Function        Remove-AzWvdAppAttachPackage                       4.2.1      Az.DesktopVirtualization
+Function        Update-AzWvdAppAttachPackage                       4.2.1      Az.DesktopVirtualization
```

Comment:
Please can you indent this to the same level as the line above (it needs to be 3 spaces only)? I don't think it'll render properly without it.

Reply:
You are correct. Updated and double checked that it rendered after the update